vpd-tool/include/tool_json_utility.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <nlohmann/json.hpp>
+
+#include <fstream>
+
+namespace vpd
+{
+namespace jsonUtility
+{
+
+/**
+ * @brief API to parse respective JSON.
+ *
+ * Exception is thrown in case of JSON parse error.
```

Comment:
suggestion: Not needed. @throw states that.

Reply:
This method along with doxygen was copied over as is from [utility/json_utility.hpp](https://github.com/ibm-openbmc/openpower-vpd-parser/blob/P11_Dev/include/utility/json_utility.hpp)

Reply:
Hmm, then we need to fix there as well :) but may be in another PR. Anyways I can live with these.